pages/common/xml-edit.md

```diff
@@ -0,0 +1,28 @@
+# xml edit
+
+> Edit an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Delete elements matching an XPATH from an XML document:
+
+`xml edit --delete {{"XPATH1"}} {{path/to/input.xml|URI}}`
```

Comment:
```suggestion
`xml edit --delete "{{XPATH1}}" {{path/to/input.xml|URI}}`
```
This applies to the whole page. It makes clear that the `" "` is required.

Reply:
Actually they're usually not. XPATH parse like filenames. I put the quotes inside because someone else required that change before. However, they're a good idea and I'll change it for you.

Reply:
Ok, I got all of them. Note that this is a direct reversal of another requested change. This puts them back they way I wrote them originally.

❯ grep '"{{' xml-*.md
xml-edit.md:`xml edit --delete "{{XPATH1}}" {{path/to/input.xml|URI}}`
xml-edit.md:`xml edit --move "{{XPATH1}}" "{{XPATH2}}" {{path/to/input.xml|URI}}`
xml-edit.md:`xml edit --rename "{{//*/@id}}" -v "{{ID}}" {{path/to/input.xml|URI}}`
xml-edit.md:`xml edit --rename "{{/xml/table/rec}}" -v "{{record}}" {{path/to/input.xml|URI}}`
xml-edit.md:`xml edit --update "{{xml/table/rec[@id=3]/@id}}" -v {{5}} {{path/to/input.xml|URI}}`
xml-escape.md:`xml escape "{{<a1>}}"`
xml-escape.md:`echo  "{{<a1>}}" | xml escape`
xml-select.md:`xml select --template --match "{{XPATH1}}" --value-of "{{XPATH2}}" {{path/to/input.xml|URI}}`
xml-select.md:`xml select --text --template --match "{{XPATH1}}" --value-of "{{XPATH2}}" --nl {{path/to/input.xml|URI}}`
xml-transform.md:`xml transform {{path/to/stylesheet.xsl}} -p "{{Count='count(/xml/table/rec)'}}" -s "{{Text="Count=}}" {{path/to/input.xml|URI}}`
xml-unescape.md:`xml unescape "{{&lt;a1&gt;}}"`
xml-unescape.md:`echo  "{{&lt;a1&gt;}}" | xml unescape`